### ISSUE TITLE
Icon alignment in problems tab

### ISSUE THREAD

**Issue by philipstarkey** [Microsoft/vscode#58777]
- VSCode Version: Started on 1.26 (still exists on 1.27.2)
- OS Version: Windows

Steps to Reproduce:

1. Install an extension which puts errors/warnings/info messages in the problems tab
2. Observe the the icons for the problems are not aligned when the problem text is long enough to be cut off and appended with an ellipsis.

Does this issue occur when all extensions are disabled?: N/A (I don't think anything puts problems in that isn't an extension??)

Here is a screenshot of the weird icon alignment. I've seen this with multiple extensions (so I don't think it's the extensions fault) and it started specifically after upgrading to 1.26 (I downgraded to 1.25 and it went away). It is also more visible if you move the problems tab to the side pane, rather than at the bottom, because you are more likely to have a line that is cut off with an ellipsis.

![image](https://user-images.githubusercontent.com/2508881/45602539-7b6bef80-ba63-11e8-83e9-b98ee5e3793f.png)

**Comment by philipstarkey** [comment]
Ah. Looks like it's because of the addition of "quick fixes" to the problems pane in 1.26. Something looks like it might be wrong with the CSS for that. When there is no quick fix for a problem (and I'm unsure it it applies if there is a quick fix as well), it seems like long messages cause the space reserved for the quick fix (actions) icon to shrink (even though the width is defined as being 16px, it shrinks to much less than that), which then results in a position offset for the error/warning/info icon.

I'm not sure exactly what/where a fix should go, but I'm guessing someone who understands that bit of code will probably be able to figure this out really quickly now :)

**Comment by sandy081** [comment]
To verify:

Create a sample extension that generates two diagnostics, one with a very long message and other with a short message like follows:

```ts
'use strict';
import * as vscode from 'vscode';

export function activate(context: vscode.ExtensionContext) {
    const diagnosticsCollection = vscode.languages.createDiagnosticCollection('test');
    const diagnostics: vscode.Diagnostic[] = [];
    diagnostics.push(new vscode.Diagnostic(new vscode.Range(new vscode.Position(1, 1), new vscode.Position(1, 10)), `Message ${0}: Arrival entered an if drawing request. How there not promotion few knowledge contented. Yet winter law behind number stairs garret excuse. Yet winter law behind number stairs garret excuse Arrival entered an if drawing request. How there not promotion few knowledge contented. Yet winter law behind number stairs garret excuse. Yet winter law behind number stairs garret excuseMessage 0: Arrival entered an if drawing request. How there not promotion few knowledge contented. Yet winter law behind number stairs garret excuse. Yet winter law behind number stairs garret excuse Arrival entered an if drawing request. How there not promotion few knowledge contented. Yet winter law behind number stairs garret excuse. Yet winter law behind number stairs garret excuse`, vscode.DiagnosticSeverity.Error));
    diagnostics.push(new vscode.Diagnostic(new vscode.Range(new vscode.Position(1, 1), new vscode.Position(1, 10)), `Message ${0}: Arrival entered an if drawing request. `, vscode.DiagnosticSeverity.Error));
    diagnosticsCollection.set(vscode.Uri.file(`/users/sandy081/work/vscode/package.json`), diagnostics);
}
```

Make sure when they are displayed without any mis-alignment in problems view
